src/sinks/redis.rs

```diff
@@ -265,12 +265,6 @@ impl EncodedLength for RedisKvEntry {
     }
 }
 
-impl ByteSizeOf for RedisKvEntry {
-    fn allocated_bytes(&self) -> usize {
-        self.key.len() + self.value.len()
-    }
-}
```

Comment:
Bonus. Do you foresee any more of these disappearing?

Reply:
I should probably do another pass and see what happens if I remove more of these, yes, good idea.